from .base_exception import BaseException

class NodeNotFoundError(BaseException):
    pass 
